documentation/endpoints/signups.md

```diff
@@ -234,3 +234,29 @@ Example response:
   }
 }
 ```
+
+## Update a Signup
+
+```
+PATCH /api/v3/signups/:signup_id
+```
+
+  - **quantity**: (int) required if why_participated is not present.
+    The number of reportback nouns verbed.
+  - **why_participated**: (string) required if quantity is not present.
+    The reason why the user participated.
+  
+Example request body:
+```
+[
+  "quantity" => "200"
+  "why_participated" => "bcuz I luv endpointz"
```

Comment:
😆

Reply:
I can't take credit for this - I saw it in another file and copy and pasted and got a giggle out of it so kept it here :) I'm glad you got a laugh out of it too!